covsirphy/cleaning/jhu_data.py

```diff
@@ -558,3 +558,64 @@ def records(self, country, province=None, start_date=None, end_date=None, popula
             raise SubsetNotFoundError(
                 country=country, country_alias=country_alias, province=province,
                 start_date=start_date, end_date=end_date, message="with 'Recovered > 0'") from None
+
+    def show_complement(self, country=None, province=None,
+                          start_date=None, end_date=None, population=None,
+                          interval=2, max_ignored=100, max_ending_unupdated=14):
+        """
+        To monitor effectivity and safety of complement on JHU subset,
+        we need to know what kind of complement was done for JHU subset
+        for each country (if country/countries specified) or for all countries.
+
+        Args:
+            country(str): country name or ISO3 code
```

Comment:
Please change to "country (str or list[str] or None): country/countries name or None (all countries)"